client/src/components/atoms/style/TextField.scss

```diff
@@ -7,14 +7,17 @@
 
   &-input {
     align-self: left;
-    border: 1px solid $input-border;
+    border: 1px solid $textfield-input-border;
     box-sizing: border-box;
     border-radius: 4px;
-    padding-left: 19px;
+    padding-left: 12px;
     line-height: 0;
+    width: 373px;
```

Comment:
I think width should be set by the user of TextField, since we may want to use TextField in different ways in the future. 

Perhaps move this styling to the div in  SearchableDropdown taht contains the TextField, and set this to width: 100% or something like that.